### ISSUE TITLE
Use https:// instead of http://

### ISSUE THREAD

**Issue by lidel** [filecoin-saturn/caboose#27]
https://github.com/filecoin-saturn/caboose/blob/93567ae43731920137b20f05a3877a10aeba6008/pool.go#L190 

> You're gonna get 301'd to https on every req

**Comment by aarshkshah1992** [comment]
Addressed in https://github.com/filecoin-saturn/caboose/pull/26.

**Comment by willscott** [comment]
won't the go client remember the hsts header and auto-upgrade after the first request?

**Comment by lidel** [comment]
no, HSTS is a browsers thing (https://github.com/golang/go/issues/10697)
